webdriver-spec.html

```diff
@@ -2670,18 +2666,27 @@
     "<code>pageLoadStrategy</code>" from <var>capabilities</var>.
 
    <li><p>If <var>strategy</var> is a string, set the <a>current session</a>’s
-    <a>page loading strategy</a> to <var>strategy</var>.
-    Otherwise, set the <a>page loading strategy</a> to <i>normal</i>.
+    <a>page loading strategy</a> to <var>strategy</var>.  Otherwise,
+    set the <a>page loading strategy</a> to <i>normal</i> and <a>set a
+    property</a> of <var>capabilities</var> with name
+    "<code>pageLoadStrategy</code>" and value "<code>normal</code>".
 
    <li><p>Let <var>proxy</var> be the result of getting property
-    "<code>proxy</code>" from <var>capabilities</var>.
-
-   <li><p>If <var>proxy</var> is a <a>proxy configuration</a> object,
-    take implementation-defined steps to set the user agent proxy
-    using the extracted <var>proxy</var> configuration.  If the
-    defined proxy cannot be configured or is not a
-    <a>proxy configuration object</a>, return <a>error</a> with
-    <a>error code</a> <a>session not created</a>.
+    "<code>proxy</code>" from <var>capabilities</var> and run the
+    substeps of the first matching statement:
+
+       <dl class=switch>
+         <dt><var>proxy</var> is a <a>proxy configuration</a> object</dt>
+         <dd><p>Take implementation-defined steps to set the user agent proxy
+             using the extracted <var>proxy</var> configuration.  If the
+             defined proxy cannot be configured return <a>error</a> with
+             <a>error code</a> <a>session not created</a>.</dd>
+
+         <dt>Otherwise</dt>
```

Comment:
We'll have deserialised as a proxy by this point --- we should never see the "otherwise" branch, and doing so is an error.